src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
> That was linked to Object.assign that remove the prototype. I wouldn't worry about this here.

Okay cool.

Now that I think about it, even if we don't implement `muiTheme` as desribed in #2509. It could make sense to make this `muiThemeable` work like this:

```js
muiTheme()(WrappedComponent);
```

That way in the future if we ever decide to add arguments that change the behavior of the HOC, we wouldn't have to go through and fix all the components again.